step-controller-plugins/pom.xml

```diff
@@ -1,40 +1,40 @@
 <!--
   Copyright (C) 2020, exense GmbH
-   
+

```

Comment:
no reformatting of existing files should be done in a PR, keep only relevant changes